packages/react-instantsearch-hooks-dom/src/ui/index.ts

```diff
@@ -0,0 +1,6 @@
+export * from './ClearRefinements';
```

Comment:
I don't really have an opinion on this, but we've been flip-flopping in the PRs, what are the pros/cons?

Pro:
- closer to what a real package will look like

Cons:
- higher chance of circular imports
- need to be careful to mark as tree-shakeable

Reply:
Is there a possibility for something other than a 1:1 direct import mapping from widget → UI (except Highlight/Snippet → Highlight)? If not, I don't think of an issue with having an index right now.

Reply:
Not much of an opinion either, but there's no _need_ for a barrel at this level so I'm fine removing it and add it only if/when we feel the pain.

Addressed in afeb265e.